scripts/emus/gba.sh

```diff
@@ -87,10 +88,11 @@ compile() {
 
 post_install() {
     echo -e "\nLinking libraries..."
-    ln -s "$INSTALL_DIR/mgba/lib/libmgba.so.0.11.0" "$INSTALL_DIR/mgba/lib/libmgba.so.0"
-    ln -s "$INSTALL_DIR/mgba/lib/libmgba.so.0.11.0" "$INSTALL_DIR/mgba/lib/libmgba.so.0.11"
+    ln -s "$INSTALL_DIR/mgba/lib/libmgba.so.0.11.0" "$INSTALL_DIR/mgba/libmgba.so.0"
+    ln -s "$INSTALL_DIR/mgba/lib/libmgba.so.0.11.0" "$INSTALL_DIR/mgba/libmgba.so.0.11"
 
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **issue (bug_risk):** Symbolic link creation seems to have a path issue, missing 'lib' directory.